### PR TITLE
Comments tests + throwing exception when not the right user is logged in

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java [hunk 1]
```diff
@@ -388,8 +388,11 @@ public void verifyUserLoggedIn(final String userName) {
         wait.forElementClickable(MERCURY_NAV_ICON);
         driver.findElement(MERCURY_NAV_ICON).click();
       } else {
-        wait.forElementVisible(By.cssSelector(
-            LOGGED_IN_USER_SELECTOR_OASIS.replace("%userName%", userName.replace(" ", "_"))));
+        WebElement avatar = driver.findElement(By.cssSelector(LOGGED_IN_USER_SELECTOR_OASIS));
+        wait.forElementVisible(avatar);
+        String loggedInUserName = avatar.getAttribute("alt");
+        if (!loggedInUserName.equals(userName))
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java]: MAJOR Missing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00121)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/commentstests/ArticleCommentsTests.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ public void ArticleComments_001_editComment() {
     editor.switchAndWrite(comment);
     article.submitComment();
     article.verifyCommentText(comment);
-    article.verifyCommentCreator(credentials.userName);
+    article.verifyCommentCreator(credentials.userNameComments);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/commentstests/ArticleCommentsTests.java]: You can use ```User.COMMENTS_REGULAR_USER.getUsername()```

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java [hunk 3]
```diff
@@ -327,7 +327,11 @@ public boolean isVideoCommentPresent(String videoName) {
   public MiniEditorComponentObject triggerEditCommentArea() {
     jsActions.scrollToElement(allCommentsArea);
     WebElement mostRecentComment = articleComments.get(0);
+    PageObjectLogging.log("First check",mostRecentComment.getText(), true);
     JavascriptExecutor js = (JavascriptExecutor) driver;
+    WebElement editButton = mostRecentComment.findElement(By.cssSelector(EDIT_BUTTON_SELECTOR));
+    Actions actions =  new Actions(driver);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java]: merge into one line

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java [hunk 4]
```diff
@@ -388,8 +388,13 @@ public void verifyUserLoggedIn(final String userName) {
         wait.forElementClickable(MERCURY_NAV_ICON);
         driver.findElement(MERCURY_NAV_ICON).click();
       } else {
-        wait.forElementVisible(By.cssSelector(
-            LOGGED_IN_USER_SELECTOR_OASIS.replace("%userName%", userName.replace(" ", "_"))));
+        WebElement avatar = driver.findElement(By.cssSelector(LOGGED_IN_USER_SELECTOR_OASIS));
+        wait.forElementVisible(avatar);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java]: no need to wait for visibility, we can wait for the presence.

File: src/test/java/com/wikia/webdriver/common/properties/Credentials.java [hunk 5]
```diff
@@ -236,6 +239,9 @@ public Credentials() {
     userNameGoSearchPreferred = XMLReader.getValue(file, "ci.user.goSearchPreferredUser.username");
     passwordGoSearchPreferred = XMLReader.getValue(file, "ci.user.goSearchPreferredUser.password");
 
+    userNameComments = XMLReader.getValue(file, "ci.user.comments.username");
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/properties/Credentials.java]: no need to add this, cause you have already ```COMMENTS_REGULAR_USER``` in Users that will provide you with both username and password for this user

**Pull request merged**